gapic/src/main/com/google/gapid/views/PipelineView.java

```diff
@@ -101,12 +112,17 @@ public PipelineView(Composite parent, Models models, Widgets widgets) {
     loading = LoadablePanel.create(this, widgets,
         panel -> createComposite(panel, new FillLayout(SWT.VERTICAL)));
 
-    stagesContainer = createComposite(loading.getContents(), new FillLayout());
+    stagesContainer = createComposite(loading.getContents(), new GridLayout(1, false));
+
+    RGB selectRGB = getDisplay().getSystemColor(SWT.COLOR_BLUE).getRGB();
+    RGB unselectRGB = getDisplay().getSystemColor(SWT.COLOR_LIST_BACKGROUND).getRGB();
+    hoverColor = new Color(getDisplay(), lerp(selectRGB, unselectRGB, 0.5f));
```

Comment:
Can I get a Windows user to do a comparison with 0.25f instead? Attached is how it looks on linux with 50% and 25%

![Screenshot from 2020-02-10 12-32-30](https://user-images.githubusercontent.com/47799839/74187504-8b104900-4c01-11ea-89b4-954f282f38b4.png)
![Screenshot from 2020-02-10 12-32-55](https://user-images.githubusercontent.com/47799839/74187510-8e0b3980-4c01-11ea-9260-e272a66a4a55.png)

Reply:
This looks different even on Linux, for example if you use a dark theme. We address the color theme issue separately, please use 0.5 for now.